proto/proto-backplane-grpc/src/main/proto/deephaven_core/proto/console.proto

```diff
@@ -121,6 +121,10 @@ message ExecuteCommandRequest {
 message ExecuteCommandResponse {
     string error_message = 1;
     io.deephaven.proto.backplane.grpc.FieldsChangeUpdate changes = 2;
+
+    // Start and end timestamps for command execution, in nanoseconds since the unix epoch.
```

Comment:
This comment is documentation for the `start_timestamp` field, not a general code comment. As such, probably makes sense to document each field as appropriate (I don't actually know if there is a way to add a "general" comment that doesn't effect resulting protobuf documentation).